src/main/java/com/devcycle/sdk/server/common/model/PlatformData.java

```diff
@@ -0,0 +1,81 @@
+package com.devcycle.sdk.server.common.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+
+public class PlatformData {
```

Comment:
can you add hostname as a param here, we pass it to AS to use as the user_id for agg events: 
<img width="501" alt="image" src="https://user-images.githubusercontent.com/1219069/191574540-05ec4f25-9e9c-4bc6-ad9e-a49b09575487.png">